src/hardware/vga_draw.cpp

```diff
@@ -2054,6 +2054,13 @@ ImageInfo setup_drawing()
 		return pcjr_or_tga();
 	};
 
+	// As far as I can tell, all Tandy modes have a height of 200.
```

Comment:
Let's be more assertive about this 😄 "All Tandy modes have a height of 200 pixels."

Reply:
fixed